audit/projects/k8s-infra-prow-build/buckets/k8s-infra-scalability-tests-logs/iam.json

```
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "members": [
+        "serviceAccount:k8s-infra-prow-build.svc.id.goog[test-pods/prow-build]"
```

@ameukam ahhhh I missed this in PR review, I'm pretty surprised this was able to be deployed... I'll create a fixup PR
```suggestion
        "serviceAccount:k8s-infra-prow-build@k8s-infra-prow-build.iam.gserviceaccount.com"
```

---

(for anyone following along at home, a workload identity binding makes no sense on a GCS bucket)

---

Opened https://github.com/kubernetes/k8s.io/pull/2239 but as I'm working through it I think this maybe makes more sense as something like `gs://k8s-infra-prow-scale-results`; I'm going to double check how the existing bucket is used by jobs and prow